mesg: ttyname failed: Inappropriate ioctl for device
cassandra bootstrap_test.py::TestBootstrap::test_simple_bootstrap_nodata false true
log_dst_dir = /root/vm_images/ZebraConf/app_meta/logs
the_test is bootstrap_test.py::TestBootstrap::test_simple_bootstrap_nodata
rm: cannot remove '/tmp/my_log.txt': No such file or directory
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-3.6.4, py-1.11.0, pluggy-0.7.1 -- /root/dtest/bin/python
cachedir: .pytest_cache
rootdir: /root/cassandra-dtest, inifile: pytest.ini
plugins: flaky-3.7.0, timeout-1.4.2, repeat-0.9.1
timeout: 1200.0s
timeout method: signal
timeout func_only: False
collecting ... collected 1 item

bootstrap_test.py::TestBootstrap::test_simple_bootstrap_nodata PASSED    [100%]
===Flaky Test Report===

test_simple_bootstrap_nodata passed 1 out of the required 1 times. Success!

===End Flaky Test Report===

========================== 1 passed in 99.76 seconds ===========================
[msx] rc = 0
mv /tmp/my_log.txt /root/vm_images/ZebraConf/app_meta/logs/bootstrap_test.py::TestBootstrap::test_simple_bootstrap_nodata-output_1644050488067_1308124777.txt
